.env.example

```diff
@@ -13,8 +13,7 @@ SESSION_SECRET=
 SENDGRID_API_KEY=
 SENDGRID_EMAIL=
 SENDGRID_ACTIVATION_TEMPLATE=
-SENDGRID_ACTIVITY_TEMPLATE=

```

Comment:
Replace all these

Reply:
Alright. Fixed. I also used the opportunity to add more tests to bookmark, update and reset password

Reply:
Thanks